subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -437,6 +437,14 @@ config BT_CTLR_ADV_RESERVE_MAX
 	  corresponding to the Advertising Data present at the time of the
 	  start/enable of Advertising is used.
 
+config BT_CTLR_ADV_AUX_SLOT_WINDOW
+	bool "Reschedule Extended Advertising Auxiliary PDUs"
```

Comment:
Is this ok?

```suggestion
	bool "Reschedule Extended Advertising Auxiliary PDUs within a slot window range"
```

Reply:
The name `BT_CTLR_ADV_AUX_SLOT_WINDOW` does not match "reschedule" IMO. From someone not as familiar with the controller as you, the description and help text looks more like an accidental copy-and-paste from a different Kconfig, than a description of this Kconfig. 

Perhaps `resched` should be part of the name?